modules/doc/content/application_usage/ondemand.md

```diff
@@ -0,0 +1,72 @@
+
+## Submission and Job Information
+
+### Specifying a Project
+Every job submission is required to have a project specified. This is normally entered with the PBS "-P" option. However, you do not need to enter "-P" here, just the project name. Project names must be selected from the list on [HPC Web](http://hpcweb.hpc.inl.gov/home/pbs#specifying-a-project). If more than one project name is appropriate, use your best judgment to choose the most applicable project name. If no project name is available that is applicable to your job, use one of the general-purpose technical area job names, e.g., “ne_gen” for nuclear energy applications.
+
+### Input File
+This is the full path to your input file that will be submitted to the applicable executable. If your input file were to be submitted via the command line, it may look like:
+
+```bash
+app-opt **-i input.i**
+```
+
+You do not need the `app-opt -i` and you need the full path to your equivalent of `input.i`. The "Select File" button should open a dialog that will allow you to browse your INL HPC folder structure and manually select your input file to prevent typos in the full path to the input file.
+
+## Advanced Code and Input Settings
+
+### Specific Version Hash
+This option allows you to run a specific build of the herd code. To get a list of the different version hashes that are allowed here, you can open a terminal on the appropriate cluster and run `module spider APP` and you will be presented with multiple versions of each herd application. 
+
+To get more information about each build, put `--help` or `-h` in the advanced parameters field below. Using the `--help` or `-h` flag will print the command help information to your output file and return.
+
+### Advanced Parameters
+By default, your submission of this forms creates a basic submission command that gets submitted to the requested 
+cluster.
+
+```bash
+app-opt -i /home/user/input.i
+```
+
+This form field allows you to add additional parameters. For example, if you wish to run performance logging, you can add a `-t` or `--timing` in this box and your application would then be executed as:
+
+```bash
+app-opt -t -i /home/user/input.i
+app-opt --timing -i /home/user/input.i
+```
+
+You are allowed to put more than one in this box. Everything that you put in this box will be entered between  `app-opt` and your input file `-i /home/user/input.i`.
+
+To get a full list and more explanation on these command line options, please see the [command line usage documentation](https://mooseframework.inl.gov/application_usage/command_line_usage.html).
+
+## Working Directory Information
+By default, your working directory is set to the same directory where your input file is located. You can change this if you would prefer a different directory by checking the checkbox and entering the new directory. You can also use the "Select Directory" button to get a file dialog box to select the directory instead of typing it out by hand.
```

Comment:
```suggestion
## Working Directory Information

By default, your working directory is set to the same directory where your input file is located. You can change this if you would prefer a different directory by checking the checkbox and entering the new directory. You can also use the "Select Directory" button to get a file dialog box to select the directory instead of typing it out by hand.
```